json/exp.json

```diff
@@ -9,8 +13,19 @@
         "verilator",
         "xgboost"
     ],
-    "experiment": "slurm_experiment",
-    "docker_home": "/<path>/<to>/slurm_home",
+    "_experiment": "experiment name",
+    "experiment": "exp",
+    "_simulation_mode": "1: execution-driven simulation w/o SimPoint, 2: trace-based simulation w/o SimPoint, 3: execution-driven simulation w/ SimPoint, 4: trace-based simulation w/ SimPoint, 5: PT-trace-based simulation w/ SimPoint",
+    "simulation_mode": "4",
+    "_workload_manager": "manual: run each exp scenario with all the simpoints at a time, slurm: Slurm schedules simulations based on its policy",
+    "workload_manager": "manual",
+    "_root_dir": "/<path>/<to>/<docker_home>",
+    "root_dir": "/home/$USER/allbench_home",
+    "_scarab_path": "/<path>/<to>/<scarab>/<repo>",
```

Comment:
rephrase: "The path to the scarab repository that contains the scarab binary used to run the experiment."